h2o-parsers/h2o-parquet-parser/src/main/java/water/parser/parquet/ParquetParser.java

```diff
@@ -201,13 +211,16 @@ ParquetParseSetup toParseSetup(byte[] parquetMetadata) {
 
   public static class ParquetParseSetup extends ParseSetup {
     transient byte[] parquetMetadata;
-
+    
```

Comment:
nit: Extra tab, the change is not needed.

Reply:
Done.